projects/templates/projects/base.html

```diff
@@ -5,6 +5,9 @@
     <title>{% block title %}TAIPAN{% endblock %}</title>
     {% load static %}
     <link rel="stylesheet" type="text/css" href="{% static 'projects/css/style.css' %}">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+          integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```

Comment:
Come on at least take credit for it! Do you understand what they do? [Here's a good explanation](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity). Essentially it's like a  simple way of defining CORS (cross-origin resource sharing) as well as verifying that nothing is tampering with your CDN code.